### PR TITLE
Add missing resources acceptance tests 

### PR DESCRIPTION
**Pull request by niuzhenguo**
adds tests for flexibleengine_networking_router_route_v2

Partially address #5

### CODE REVIEW

**Comment by niuzhenguo** [comment]
please hold on for a while, will also add AS resources tests with this PR.

**Comment by niuzhenguo** [comment]
@freesky-edward @edisonxiang please have a look at this when you got time, thanks!

File: flexibleengine/resource_flexibleengine_as_group_v1_test.go [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+		found, err := groups.Get(asClient, rs.Primary.ID).Extract()
+		if err != nil {
+			return err
+		}
+
+		if found.ID != rs.Primary.ID {
+			return fmt.Errorf("Autoscaling Group not found")
+		}
+		log.Printf("[DEBUG] test found is: %#v", found)
+		group = &found
```
Comment (edisonxiang) [inline flexibleengine/resource_flexibleengine_as_group_v1_test.go]: I am not sure whether we should use `*group= *found` here.

Comment (niuzhenguo) [reply flexibleengine/resource_flexibleengine_as_group_v1_test.go]: It's a common manner for such test, just use to check the returned object and the specified groups.Group with the same type.

Comment (edisonxiang) [reply flexibleengine/resource_flexibleengine_as_group_v1_test.go]: ack

**Comment by freesky-edward** [comment]
merged, thanks

**Pull request merged**